File: naisc-rest/src/main/java/org/insightcentre/uld/naisc/rest/InvalidConfigurationException.java
package org.insightcentre.uld.naisc.rest;

public class InvalidConfigurationException extends Exception {

    public InvalidConfigurationException() {
    }

    public InvalidConfigurationException(String s) {
        super(s);
    }

    public InvalidConfigurationException(String s, Throwable throwable) {
        super(s, throwable);
    }

    public InvalidConfigurationException(Throwable throwable) {
        super(throwable);
    }

    public InvalidConfigurationException(String s, Throwable throwable, boolean b, boolean b1) {
        super(s, throwable, b, b1);
    }
}

File: naisc-rest/src/main/kotlin/org/insightcentre/uld/naisc/rest/InvalidConfigurationException.kt
package org.insightcentre.uld.naisc.rest

class InvalidConfigurationException : Exception {

    constructor() {}

    constructor(s: String) : super(s) {}

    constructor(s: String, throwable: Throwable) : super(s, throwable) {}

    constructor(throwable: Throwable) : super(throwable) {}

    constructor(s: String, throwable: Throwable, b: Boolean, b1: Boolean) : super(s, throwable, b, b1) {}
}
